packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingMethods.js

```diff
@@ -34,7 +32,7 @@ const ShippingMethods = props => {
         selectedShippingMethod,
         shippingMethods,
         showForm
-    } = useShippingMethods({ ...ShippingMethodsOperations });
+    } = useShippingMethods({});
```

Comment:
Same as the other talon calls.
Remove empty object and defined default prop in talon like so (props = {}).
Will let you update each calls and review after.